The `image` feature no longer compiles in combination with the `glow` feature

### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Is this issue related to iced?

- [X] My hardware is compatible and my graphics drivers are up-to-date.

### What happened?

This was introduced in #1578

### What is the expected behavior?

The crate compiles 

### Version

master

### Operative System

Windows

### Do you have any log output?

```shell
Compiling iced_glow v0.5.0 (D:\source\rust\iced\glow)
error[E0027]: pattern does not mention field `color`
   --> glow\src\image.rs:195:17
    |
195 |                 layer::Image::Vector { handle: _, bounds } => (None, bounds),
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing field `color`
    |
help: include the missing field in the pattern
    |
195 |                 layer::Image::Vector { handle: _, bounds, color } => (None, bounds),
    |                                                         ~~~~~~~~~
help: if you don't care about this missing field, you can explicitly ignore it
    |
195 |                 layer::Image::Vector { handle: _, bounds, .. } => (None, bounds),
```

--- a/glow/src/image.rs
+++ b/glow/src/image.rs
@@ -192,7 +192,7 @@ impl Pipeline {
                 }
 
                 #[cfg(not(feature = "svg"))]
-                layer::Image::Vector { handle: _, bounds } => (None, bounds),
+                layer::Image::Vector { bounds, .. } => (None, bounds),
             };
 
             unsafe {

